scripts/Connector/Connector_Hirose/conn_hirose_df63_tht_top.py

```diff
@@ -149,7 +149,11 @@ def generate_one_footprint(pins, configuration):
     )
 
     #mounting hole
-    kicad_mod.append(Pad(at=[-1.5,3.25],type=Pad.TYPE_NPTH,layers=Pad.LAYERS_NPTH,shape=Pad.SHAPE_CIRCLE,size=mount_size,drill=mount_size))
+    if form_type != 'R':
```

Comment:
Since the M version is being specifically implemented, can you change this to `==`? That is more clear to read.